recurly/resource.py

```diff
@@ -445,11 +445,23 @@ def value_for_element(cls, elem):
                 return value_class.from_element(elem)
 
         # Untyped complex elements should still be resource instances. Guess from the nodename.
-        if len(elem):  # has children
+        if len(elem) == 1:
             value_class = cls._subclass_for_nodename(elem.tag)
             log.debug("Converting %r tag into a %s", elem.tag, value_class.__name__)
             return value_class.from_element(elem)
 
+        # Untyped complex elements should still be resource instances. Guess from the nodename.
```

Comment:
Maybe delete this duplicated comment and say something to the effect that we de-serialize ramp intervals when type="array" is not provided, i.e. with ramp_intervals.